pandas/_libs/tslibs/timezones.pyx

```diff
@@ -111,16 +111,16 @@ cpdef inline object maybe_get_tz(object tz):
     return tz
 
 
-def _p_tz_cache_key(tz):
+def p_tz_cache_key(tz):
     """ Python interface for cache function to facilitate testing."""
```

Comment:
this is dumb that we are exposing a function just to test. I don't think this is actually necessary and should be removed (so maybe leave this one as private).

Reply:
Yah, best guess is that was created before `cpdef` was an option.  Will re-privatize.